app_dart/lib/src/request_handlers/github_webhook.dart

```diff
@@ -187,6 +196,8 @@ class GithubWebhook extends RequestHandler<Body> {
       commitSha: pr.head.sha,
       repositoryName: pr.head.repo.name,
     );
+    await githubStatusService.setBuildsPendingStatus(
```

Comment:
It's better to pass in the slug with `pr.base.repo.slug()` instead of synthesizing it in the function.

Reply:
Done